src/msw/headerctrl.cpp

```diff
@@ -386,6 +386,27 @@ void wxHeaderCtrl::DoInsertItem(const wxHeaderColumn& col, unsigned int idx)
     {
         wxLogLastError(wxT("Header_InsertItem()"));
     }
+
+    // Per-column IsResizable() cannot be accurately implemented, the resizing
```

Comment:
This is not quite accurate, it is implemented when we handle `HDN_BEGINTRACK`, but it's true that the cursor still changes to the resize one in this case, i.e. I understand what you mean here, but another person reading it could misunderstand this and decide that columns can still be resized even when they're marked as not resizeable, which is not the case.